tests/integration_tests/data_steward/validation/participants/validate_test.py

```diff
@@ -0,0 +1,177 @@
+"""
+Integration test for the validate_email module
+
+Ensures that emails are correctly identified as matches and non-matches
+between EHR and RDR.
+"""
+
+# Python imports
+import os
+from unittest import TestCase
+
+# Third party imports
+from google.cloud.bigquery import DatasetReference
+
+# Project imports
+import bq_utils
+from utils import bq
+from tests import test_util
+from app_identity import PROJECT_ID
+from common import JINJA_ENV, DRC_OPS, PS_API_VALUES
+from validation.participants.validate import identify_rdr_ehr_match
+from constants.validation.participants.identity_match import IDENTITY_MATCH_TABLE
+
+POPULATE_PS_VALUES = JINJA_ENV.from_string("""
+INSERT INTO `{{project_id}}.{{drc_dataset_id}}.{{ps_values_table_id}}` 
+(person_id, email)
+VALUES 
+    (1, 'john@gmail.com'),
+    (2, 'rebecca@gmail.com'),
+    (3, 'samwjeo'),
+    (4,'chris@gmail.com')
+""")
+
+POPULATE_ID_MATCH = JINJA_ENV.from_string("""
+INSERT INTO `{{project_id}}.{{drc_dataset_id}}.{{id_match_table_id}}` 
+(person_id, email, algorithm)
+VALUES 
+    (1, 'missing_ehr', 'no'),
+    (2, 'missing_ehr', 'no'),
+    (3, 'missing_ehr', 'no'),
+    (4,'missing_ehr', 'no')
+""")
+
+ID_MATCH_CONTENT_QUERY = JINJA_ENV.from_string("""
+    SELECT
+        *
+    FROM `{{project_id}}.{{drc_dataset_id}}.{{id_match_table_id}}`
+""")
+
+
+class ValidateTest(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = os.environ.get(PROJECT_ID)
+        self.dataset_id = os.environ.get('COMBINED_DATASET_ID')
+        self.dataset_ref = DatasetReference(self.project_id, self.dataset_id)
+        self.client = bq.get_client(self.project_id)
+
+        self.hpo_id = 'fake_site'
+        self.id_match_table_id = f'{IDENTITY_MATCH_TABLE}_{self.hpo_id}'
+        self.ps_values_table_id = f'{PS_API_VALUES}_{self.hpo_id}'
+        self.pii_email_table_id = f'{self.hpo_id}_pii_email'
+
+        # Create and populate the ps_values site table
+
+        ps_values_fields = [
+            dict(name='person_id', type='integer', mode='nullable'),
+            dict(name='email', type='string', mode='nullable'),
+            dict(name='algorithm', type='string', mode='nullable')
+        ]
+
+        bq_utils.create_table(self.ps_values_table_id,
+                              ps_values_fields,
+                              drop_existing=True,
+                              dataset_id=self.dataset_id)
```

Comment:
nit: Standard table schemas can sometimes prevent differences between test and prod behavior.
The following needs a couple of imports - fields_for from resources and Table from bigquery.
```suggestion
        schema = resources.fields_for(f'{PS_API_VALUES}')
        table = Table(f'{self.project_id}.{self.dataset_id}.{self.ps_values_table_id}', schema=schema)
        table = self.client.create_table(table)
```

Reply:
Just pushed some changes to address these. In the process, I modified create_table statements to create partitions during testing.

Reply:
Amazing, that mimics prod way better, thanks for doing this!